pkg/constants/constants.go

```diff
@@ -97,6 +97,9 @@ const (
 	// OSMCertificateValidityPeriod is the TTL of the certificates used in the OSM control plane or for Envoy to xDS communication.
 	OSMCertificateValidityPeriod = 87600 * time.Hour // a decade
 
+	// CertCheckInterval is the period of ticks for a Ticker
+	CertCheckInterval = 5 * time.Second
```

Comment:
5 seconds feels a little aggressive for this check, thoughts? I guess it is just checking in the informer cache, but still we can probably get away with reducing this.

Particularly if we change the remaining time till expiration in shouldRotate https://github.com/openservicemesh/osm/issues/5150

Reply:
s/reducing/increasing?